src/main/resources/config/application.yml

```diff
@@ -0,0 +1,67 @@
+# All env variables that are used in one place
+# Exception: TERRA_ENV (which sets the active profile), which is parsed before this is.
+env:
```

Comment:
[aside] I'm still coming to terms with the idea of this "env" style of mapping env variables to Spring app config. It's attractive in some regards, but I think it also has some potential to create more confusion.

Concrete example: with this PR, there are now 3 totally different ways of referring to the "config value that tells WSM whether to initialize its database on startup":
1) `INIT_DB` env variable
2) `env.db.init` Spring config
3) `workspace.db.workspace.initializeOnStart` Spring config

IMO, we should eventually aim for there to be one canonical path for each config value, no matter where it's getting specified.

I don't think this is any reason to hold off on this PR. But I'd just note that I suspect we may eventually seek a solution that doesn't require a separate 'env' stanza like this.